new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/ExecCommand.java

```diff
@@ -88,14 +83,17 @@ public static ExecResult exec(
       }
 
       p.waitFor();
-      
-      // if we have not read the stdout, we do it now
-      if (stdout == null) {
-        stdout = read(in.getInputStream());
+
```

Comment:
curiously, i don't see this "remove code" in develop at all -- you may need to rebase or pull develop into your branch

Reply:
I checked and the code is still there in develop branch. I am wondering if you were looking at a same-named class in the old integration test code.

Reply:
oh i was. sorry, my mistake - the review view truncates the file name and i assumed :( :( :(